### ISSUE TITLE
Latest version not running under new install of windows 10

### ISSUE THREAD

**Issue by MarkusRissmann** [gitextensions/gitextensions#7726]
I have a brand new install of windows 10 in VMWare workstations.

I am installing "GitExtensions-3.3.1.7897.msi".

After installing I launch GitExtension from the desktop shortcut. Nothing appears to happen. If I view in Task Manager I see "Git Extentions" being launched 20-30+ times and growing.

I had to disable windows defender to get it to run in the first place else I would get a message saying git exe not found.

**Comment by RussKie** [comment]
Hmm that's nasty.
If git isn't present the app should show a popup saying just that.

**Comment by MarkusRissmann** [comment]
ahhh, so that is what it was telling me. The message wasn't clear at all. This is a new environment so I was trying to install minimal apps. I guess to minimal.

**Comment by RussKie** [comment]
How would you reword the message to be clearer?

**Comment by MarkusRissmann** [comment]
something like 

"Please make sure that the GIT library has been installed" 

or more verbose  

"GIT is a prerequisite of GitExtentions. Please make sure it is installed". 

My confusion was that I was already having issues installing GitExtensions with windows defender so I thought the error had to do with the current installation not a missing resource,

**Comment by RussKie** [comment]
Is it not what you saw?
![image](https://user-images.githubusercontent.com/4403806/73831061-d0c3bf00-4859-11ea-84b7-1a976a492cdc.png)

**Comment by MarkusRissmann** [comment]
Yes, but as I said, I was confused and thought something was wrong with the current installation of GitExtensions because of the previous warnings and blocking from windows defender. It didn't dawn on me (but is very apparent now) that this error was referring to a prerequisite.

**Comment by RussKie** [comment]
@gitextensions/git-extensions-team thoughts on the proposed wording?

**Comment by NikolayXHD** [comment]
The Git executable could not be located on your system, Git is a prerequisite to GitExtensions.

**Comment by MarkusRissmann** [comment]
Sounds good! Thx

**Comment by JonathanLindsey** [comment]
In my opinion the words 'located' and 'find' as used in the current dialog are confusing. I think it should be worded something like this:

> Git Extensions requires a Git executable to be on your computer which does not appear to be the case. Please install Git or click Browse to find Git on your computer manually.
> 
> **Install Git**
> _Go to git-scm.com_
> 
> **Browse to Git**
> _Find Git on your computer_
> 
> More info

The **bolded lines** should be large text and the _italicized lines_ should be smaller text on the same button.

The "More info" should be small text and be a [link to here](https://github.com/gitextensions/gitextensions/wiki/Application-Dependencies#git) which is what the current "Install Git..." button links to.

**Comment by RussKie** [comment]
> More info

TaskDialog doesn't support this.



> **Install Git**
> _Go to git-scm.com_

No, it must go to our wiki.

I also don't think links should be re-ordered. 

Other than this, extending the text is a good idea, it makes the dialog more user friendly. 
A pull-request is welcome.

**Comment by hoborm** [comment]
I have used the wording suggested by @JonathanLindsey, However @RussKie gave his own feedback in #8519 Can anyone point me to the right direction on which suggestions should I take into account or even if these changes are still necessary altogether? I just tried to pick up some up-for-grabs issues in my free time but I do not want the PR to get stalled.

**Comment by RussKie** [comment]
Let's action the feedback I left you on the issue.

**Comment by JonathanLindsey** [comment]
@RussKie Please reread the first 7 posts on this issue. MarkusRissmann made the point that "It didn't dawn on me (but is very apparent now) that this error was referring to a prerequisite."

Additionally, what is confusing to me with the old dialog is that when I click "Install Git..." I am taken to https://github.com/gitextensions/gitextensions/wiki/Application-Dependencies#git which looks quite scary and the link to git-scm.com does not stand out very well even though it is near the top of the page. That is why I suggested making the "Install Git..." button take users directly to the git-scm.com download page. The more info button ("Go to GitExtensions wiki" is fine) being a separate button could also be good in case someone needs more information.

I believe the current implementation in #8519 with the old error message is simply adding descriptions for the buttons. I think that is good but won't actually fix the main issue.
